lib/controllers/file-patch-controller.js

```diff
@@ -172,8 +172,12 @@ export default class FilePatchController extends React.Component {
       {oldMode: fp.getOldMode(), newMode: fp.getNewMode()} :
       null;
     const symlinkChange = fp && fp.hasSymlink() ?
-      {oldSymlink: fp.getOldFile().getSymlink(), newSymlink: fp.getNewFile().getSymlink(), status: fp.getStatus()} :
-      null;
+    {
+      oldSymlink: fp.getOldFile().getSymlink(),
+      newSymlink: fp.getNewFile().getSymlink(),
```

Comment:
Can we throw these on `FilePatch` directly, similar to `getOldPath` and `getNewPath`?